samaSiriwardhanage/spotlights
# #1 update cta text

Resolves #1 

I would like to indicate that visitors can select videos by clicking arrows to the next screen, and either wait for the next video to play physically move to adjacent Spotlights.

### Acceptance Criteria

- [ ] Update call to action text to TAP LENS ON READER 

### Relevant design files
https://www.figma.com/file/r8dJf66KcjkRONvMTqd8rF/VE---Gallery?node-id=729%3A13076

### Testing instructions
Add a list of steps required to test the feature.
1. See the text that matches to the design.
